website/docs/r/registry_module.html.markdown

```diff
@@ -77,14 +77,21 @@ resource "tfe_organization" "test-organization" {
   email = "admin@company.com"
 }
 
-resource "tfe_registry_module" "test-no-code-provisioning-registry-module" {
+resource "tfe_registry_module" "test-ncp-registry-module" {
   organization    = tfe_organization.test-organization.name
   namespace       = "terraform-aws-modules"
   module_provider = "aws"
   name            = "vpc"
   registry_name   = "public"
   no_code         = true
 }
+
+resource "tfe_nocode_module" "foobar" {
```

Comment:
this doc is for the `tfe_registry_module`, the `"tfe_nocode_module"` resource example should go into it's own doc, is there a reason for adding it here too?